zulipterminal/model.py

```diff
@@ -141,6 +141,9 @@ def is_search_narrow(self) -> bool:
         """
         return 'search' in [subnarrow[0] for subnarrow in self.narrow]
 
+    def set_found_newest(self, found_newest: bool) -> None:
+        self.found_newest = found_newest
```

Comment:
When requiring a method to access data, or just hiding it in an object/class, I find it useful to have an underscore prefix. That both gives a visual indication elsewhere if we're touching something we shouldn't be (ie. potentially hacky code), but also in making this kind of transition it's a variable rename, so anything that depended upon `Model.found_newest` would fail with a name error.